package.json

```diff
@@ -1,17 +1,11 @@
 {
   "name": "sanitize-html",
-  "version": "1.27.1",
+  "version": "2.0.0-beta",
   "description": "Clean up user-submitted HTML, preserving whitelisted elements and whitelisted attributes on a per-element basis",
   "sideEffects": false,
-  "main": "dist/sanitize-html.js",
-  "files": [
-    "dist/"
-  ],
+  "main": "index.js",
   "scripts": {
-    "build": "mkdir -p dist && browserify src/index.js > dist/sanitize-html-es2015.js --standalone 'sanitizeHtml' && babel dist/sanitize-html-es2015.js --out-file dist/sanitize-html.js --presets=@babel/preset-env",
-    "minify": "npm run build && uglifyjs dist/sanitize-html.js > dist/sanitize-html.min.js",
-    "prepublishOnly": "npm run minify",
-    "test": "npx eslint . && npm run prepublishOnly && mocha test/test.js"
+    "test": "npx eslint . && mocha test/test.js"
```

Comment:
IMO better put lint after tests caz it produces less important for correctness errors.

Reply:
Fair point.